.github/workflows/ci-pytorch_test-conda.yml

```diff
@@ -18,16 +18,16 @@ defaults:
 jobs:
   conda:
     runs-on: ubuntu-20.04
-    container: pytorchlightning/pytorch_lightning:base-conda-py${{ matrix.python-version }}-torch${{ matrix.pytorch-version }}
+    container: pytorchlightning/pytorch_lightning:base-conda-py${{ matrix.python-version }}-torch${{ matrix.pytorch-version }}-cuda${{ matrix.cuda-version }}
     strategy:
       fail-fast: false
       matrix:
         # nightly: add when there's a release candidate
         include:
-          - {python-version: "3.8", pytorch-version: "1.9"}
-          - {python-version: "3.8", pytorch-version: "1.10"}
-          - {python-version: "3.9", pytorch-version: "1.11"}
-          - {python-version: "3.9", pytorch-version: "1.12"}
+          - {python-version: "3.8", pytorch-version: "1.9", cuda-version: "11.1"}
+          - {python-version: "3.8", pytorch-version: "1.10", cuda-version: "11.1"}
```

Comment:
cuda-version: "11.1" -> "11.1.1" to keep consistent with below changes

Reply:
Thanks for your review! Still wip!